include/libultraship/libultra/interrupt.h

```diff
@@ -1,6 +1,5 @@
-#ifndef ULTRA64_INTERRUPT_H
-#define ULTRA64_INTERRUPT_H
+#pragma once
 
-typedef u32 OSIntMask;
+#include <stdint.h>
```

Comment:
why is this being added?

Reply:
This was also causing clang-tidy to error